### ISSUE TITLE
main branch Windows build crash

### ISSUE THREAD

**Issue by JunkuiZhang** [zed-industries/zed#9111]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Clone the main branch, then `cargo run`, everything works fine.

But, if I run `Zed.exe` directly from the `debug` dir, it will crash.

### Environment

windows 11

**Comment by JunkuiZhang** [comment]
Error: 
```rust
Thread "main" panicked with "Could not start watcher on parent directory: Error { kind: Generic(\"Input watch path is neither a file nor a directory.\"), paths: [] }" at crates\fs\src/fs.rs:398:14
```

I tested and debug prints something. When the `path` is ` crates/zed/src/languages`, it crashed.

Inside `notify` crate, it gets dir: `C:\zjk\projects\zed\target\debug\crates/zed/src/languages`, which just dose not exist.

**Comment by JunkuiZhang** [comment]
Will be fixed by #9112

**Comment by Moshyfawn** [comment]
Has this been resolved by #9112?

**Comment by JunkuiZhang** [comment]
> Has this been resolved by #9112?

Yes, sorry I forgot to close this.
